test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ViewComponentTests.cs

```diff
@@ -140,6 +140,63 @@ public void ViewComponent_View_WithModelParameter_SetsResultViewWithDefaultViewN
         }
 
         [Fact]
+        public void ViewComponent_View_WithNullModelParameter_SetsResultViewWithDefaultViewNameAndNullModel()
+        {
+            // Arrange
+            var viewComponent = new TestViewComponent();
+            viewComponent.ViewData.Model = new object();
+
+            // Act
+            var actualResult = viewComponent.View(model: null as object);
```

Comment:
(BTW the `var` in this case resolves to `object`, but using `var` isn't my point, but rather that the variable is declared (implicitly or explicitly) as `object`.)